### ISSUE TITLE
How to deal with 'NAN' in multi label task ?

### ISSUE THREAD

**Issue by ykk648** [open-mmlab/mmclassification#631]
For example, I have 1 picture with 3 classes, label is [0,1,nan], by using `MultiLabelLinearClsHead` with `CrossEntropyLoss`, is there any existed way to make grad of label `nan` do not backpropagate?

**Comment by ykk648** [comment]
I found `class_weight` param in definition of `CrossEntropyLoss`, so the solution is name `class_weight` when training with 'NAN' label data, but how to mix `NAN` and normal data in consideration of loss was bind with model but not dataset?

**Comment by Ezra-Yu** [comment]
Why is there a category of `nan`, this is unreasonable. 

If you don’t perform backpropagation, you can’t predict it. Why is this label designed?

**Comment by ykk648** [comment]
@Ezra-Yu   Some images were not labeled well, we only know real class-1 and class-2 but not class-3, in case of mini-dataset, this kind of situation may occur.

**Comment by Ezra-Yu** [comment]
In other words, there is a missing label. The `class_weight` may not meet your requirements. If the coefficient of one of the categories is set to 0, then this category cannot be predicted correctly

I think you can implement your own loss and set the nan value to 0 before averaging. refer to the [documtation](https://mmclassification.readthedocs.io/en/latest/tutorials/new_modules.html).

**Comment by ykk648** [comment]
thanks, issue close.
